heron/api/tests/java/com/twitter/heron/streamlet/impl/utils/StreamletUtilsTest.java

```diff
@@ -0,0 +1,32 @@
+//  Copyright 2017 Twitter. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+package com.twitter.heron.streamlet.impl.utils;
+
+import org.junit.Test;
+
+public class StreamletUtilsTest {
```

Comment:
Related to my comment on the `StreamletUtils` class above, I'd say include this with the `StreamletImplTest` for now. We can create a separate test suite for streamlet utils at a future time when a separate utils class makes more sense. The test is a good idea, though.

Reply:
UT coverages have been addressed for two use cases via ```testSetNameWithInvalidValues``` and ```testSetNumPartitionsWithInvalidValue``` and also ```require``` is set as private to avoid opening to public.